salesagility/SuiteCRM
# Fixed Issue 3010 - Cannot import email when using Theme P

## Description




Removing unnecessary z-index styles.

## Motivation and Context


Mask z-index was too high.

## How To Test This


Navigate to Emails Module and Click on Settings/Add Account, Add Outbound and these dialog should be accessible by the user above the mask.
Also in Admin > Email Settings Sent Test Email.

## Types of changes


### Final checklist
